chain/chain.go

```diff
@@ -17,7 +17,7 @@ var (
 	GenesisGasLimit uint64 = 4712388
 
 	// GenesisDifficulty is the default difficulty of the Genesis block.
-	GenesisDifficulty = big.NewInt(131072)
+	GenesisDifficulty = big.NewInt(1)
```

Comment:
Have we confirmed that modifying this value will cause the breaking change, and leaving it alone will not?

Reply:
Yes, I have tested the change with a local 4 validator node setup under this build. I updated the PR description on that matter.